src/diffusers/models/autoencoders/autoencoder_kl_cogvideox.py

```diff
@@ -1182,7 +1182,8 @@ def _encode(self, x: torch.Tensor) -> torch.Tensor:
 
         frame_batch_size = self.num_sample_frames_batch_size
         # Note: We expect the number of frames to be either `1` or `frame_batch_size * k` or `frame_batch_size * k + 1` for some k.
-        num_batches = num_frames // frame_batch_size if num_frames > 1 else 1
+        # So, it is okay to not round up as the extra remaining frame is handled in the loop
```

Comment:
Could make this comment more formal and clearer.

Reply:
updated, WDYT now?

Reply:
Looks good!